### ISSUE TITLE
One time initialization in ``MessageHandlerMixin.add_one_message`` can trigger ``KeyError`` in some scenario

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [PyCQA/pylint#5860]
### Bug description

See discussion : https://github.com/PyCQA/pylint/commit/cbd3cc07515e21ed08000941dc4883c86e84e208#r67509745

### Pylint output

```shell
Traceback (most recent call last):
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/lint/pylinter.py", line 577, in load_plugin_configuration
    module = astroid.modutils.load_module_from_name(modname)
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/astroid/modutils.py", line 218, in load_module_from_name
    return importlib.import_module(dotted_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pylint_plugins'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/google/home/avolkov/.local/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/__init__.py", line 24, in run_pylint
    PylintRun(sys.argv[1:])
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/lint/run.py", line 359, in __init__
    linter.load_plugin_configuration()
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/lint/pylinter.py", line 581, in load_plugin_configuration
    self.add_message("bad-plugin-value", args=(modname, e), line=0)
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/message/message_handler_mix_in.py", line 244, in add_message
    self.add_one_message(
  File "/usr/local/google/home/avolkov/.local/lib/python3.9/site-packages/pylint/message/message_handler_mix_in.py", line 306, in add_one_message
    self.stats[msg_cat] += 1
KeyError: 'error'
```

### Expected behavior

No crash

### Pylint version

```shell
pylint==2.11.1
astroid 2.8.6
```

**Comment by DanielNoord** [comment]
@Ark-kun Are you able to share the relevant code you were talking about in that comment? The handling of `stats` has been completely rewritten (by your truly).
Also, what do you mean "use any other version". Are you trying to update to `2.12.x`?

**Comment by jacobtylerwalls** [comment]
Right, I think this was fixed in 16c09cb48a3c66f453faccdf380aea6969d61202. I suggest sending a pull request to the project to update their pylint version in the pre-commit hook.

**Comment by Pierre-Sassoulas** [comment]
Closing as I think the commit linked by @jacobtylerwalls fix the issue and we don't support 2.11 anymore.
